src/rgw/driver/rados/rgw_notify.cc

```diff
@@ -340,6 +341,23 @@ class Manager : public DoutPrefixProvider {
       boost::system::error_code ec;
 	    timer.async_wait(yield[ec]);
     }
+    ldpp_dout(this, 5) << "INFO: manager stopped. done cleanup for queue: " << queue_name << dendl;
+  }
+
+  // unlock (lose ownership) queue
+  int unlock_queue(const std::string& queue_name, librados::IoCtx& ioctx) {
+    const auto ret = rados::cls::lock::unlock(&ioctx, queue_name, queue_name+"_lock", lock_cookie);
```

Comment:
i don't use `auto` to save on typing, i use it to avoid implicit/unintentional casting.
and the need to lookup the return value from a function unless i have to know it.
e.g. if `rados::cls::lock::unlock()` returns `unsigned` and i keep `auto` (which would be unsiged), i would get an warning in: `if (ret == -ENOENT)`.
but if i cast to `int`, i won't get this warning.
see: https://godbolt.org/z/fa3nj4rr3